src/ios/OneSignalPush.m

```diff
@@ -415,4 +417,30 @@ - (void)pauseInAppMessages:(CDVInvokedUrlCommand*)command {
    bool pause = [command.arguments[0] boolValue];
    [OneSignal pauseInAppMessages:pause];
 }
-@end
\ No newline at end of file
+
+- (void)sendOutcome:(CDVInvokedUrlCommand*)command {
+    NSString *name = command.arguments[0];
+
+    [OneSignal sendOutcome:name onSuccess:^(OSOutcomeEvent *outcome){
+        successCallback(command.callbackId, [outcome jsonRepresentation]);
+    }];
+}
+
+- (void)sendUniqueOutcome:(CDVInvokedUrlCommand*)command {
+    NSString *name = command.arguments[0];
+
+    [OneSignal sendUniqueOutcome:name onSuccess:^(OSOutcomeEvent *outcome){
+        successCallback(command.callbackId, [outcome jsonRepresentation]);
+    }];
+}
+
+- (void)sendOutcomeWithValue:(CDVInvokedUrlCommand*)command {
+    NSString *name = command.arguments[0];
+    NSNumber *value = command.arguments[1];
+
+    [OneSignal sendOutcomeWithValue:name value:value onSuccess:^(OSOutcomeEvent *outcome){
+        successCallback(command.callbackId, [outcome jsonRepresentation]);
+    }];
```

Comment:
Not sure how well OneSignal is protected for null values. Maybe add some null checkers for name and value?

Reply:
I can confirm...